README.md

```diff
@@ -169,33 +129,24 @@ You can deploy this project using any host of your choice
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/Bigizi/OOP-school-library/issues/).
+Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project give ⭐️ and share with your friends!
+
+If you like this project please give me a star.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank @Microverse online school.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-- **[How to run ruby program]**
-
-  - [use irb on command line]
+I would like to thank Microverse which provided me all knowledge about full stack development.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] @team great work on your documentation, however, the MIT license link is not working, 
![image](https://user-images.githubusercontent.com/61408860/234689188-80aafc7d-6d68-4964-a2cd-5e6f80acd343.png)